static/app/views/dashboards/widgetBuilder/releaseWidget/fields.tsx

```diff
@@ -237,7 +244,7 @@ export const SESSIONS_OPERATIONS: Readonly<
     parameters: [
       {
         kind: 'column',
-        columnTypes: ['integer', 'string'],
```

Comment:
we dont have `unhealthy_rate(user)` support in the session api. It's also not needed for any of the insights charts.